src/components/Modal/Modal.vue

```diff
@@ -77,6 +87,11 @@ const modal = ref<HTMLElement>();
 
 const bsModal = ref();
 
+const {classes} = useClasses(computed(() => [
+    computed(() => props.modalCentered ? 'modal-dialog-centered' : null).value,
+    computed(() => props.modalSize ? useModalSize(props.modalSize).sizeClass.value : null).value,
+]));
```

Comment:
Feels like this would already be enough.
```suggestion
const {classes} = useClasses(computed(() => [
    props.modalCentered ? 'modal-dialog-centered' : null,
    props.modalSize ? useModalSize(props.modalSize).sizeClass.value : null,
]));
```